app/views/logix/examples.html.erb

```diff
@@ -1,101 +1,108 @@
+            <div class="card-footer card-footer-examplepage-example">
+              <p class="examplepage-example-p">FlipFlop</p>
+              <button type="button" class="btn btn-about examplepage-example-button" onclick="window.location.href = 'https://circuitverse.org/users/3/projects/251';">View</button>
+            </div>
+          </div>
         </div>
-        <br />
-
-        <div class="row feature">
-                 <div class="col-12 col-md-8">
-                     <h3 class="card-title">ALU 74LS181 by Ananth Shreekumar</h4>
-                     <p class="card-text"></p>
-                          <a class="btn btn-dark btn-lg" target="_blank" href="https://circuitverse.org/users/126/projects/252" role="button">View Example &raquo;</a>
-                 </div>
-                 <div class="col">
-                     <%= image_tag "/img/ALU.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
-                 </div>
+        <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4">
+          <div class="card card-examplepage-example">
+            <%= image_tag "/assets/examples/Keyboard_n.jpeg", alt: "Keyboard", :class => "card-img-top card-img-examplepage-example" %>
+            <div class="card-footer card-footer-examplepage-example">
+              <p class="examplepage-example-p">Keyboard</p>
+              <button type="button" class="btn btn-about examplepage-example-button" onclick="window.location.href = 'https://circuitverse.org/users/3/projects/250';">View</button>
+            </div>
+          </div>
         </div>
-        <br />
-
-        <div class="row feature">
-                 <div class="col-12 col-md-8">
-                     <h3 class="card-title">SAP-1</h4>
-                     <p class="card-text">SAP-1 short for simple as possible computer is a 8 Bit computer. It can perform simple operations like Addition and Subtraction.</p>
-                          <a class="btn btn-dark btn-lg" target="_blank"  href="https://circuitverse.org/users/3/projects/67" role="button">View Example &raquo;</a>
-                 </div>
-                 <div class="col">
-                     <%= image_tag "/img/SAP.jpeg", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
-                 </div>
+        <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4">
+          <div class="card card-examplepage-example">
+            <%= image_tag "/assets/examples/ALU_n.png", alt: "ALU", :class => "card-img-top card-img-examplepage-example" %>
+            <div class="card-footer card-footer-examplepage-example">
+              <p class="examplepage-example-p">ALU 74LS181 by Ananth Shreekumar</p>
+              <button type="button" class="btn btn-about examplepage-example-button" onclick="window.location.href = 'https://circuitverse.org/users/3/projects/252';">View</button>
+            </div>
+          </div>
```

Comment:
Store the examples in a loop and iterate over them instead of hard coding them separately 
```ruby
examples = [{
name: ''
link: ''
}, {name: '', link: ''}]

examples.each do |example|
end
```

Something like this